src/Akka.Streams.Kafka/Stages/Consumers/Committers.cs

```diff
@@ -0,0 +1,35 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Confluent.Kafka;
+
+namespace Akka.Streams.Kafka.Stages.Consumers
+{
+    /// <summary>
+    /// Interface for implementing committing consumed messages
+    /// </summary>
+    internal interface IInternalCommitter 
+    {
+        /// <summary>
+        /// Commit all offsets (of different topics) belonging to the same stage
+        /// </summary>
+        Task Commit();
```

Comment:
@Aaronontheweb Alpakka returnes `Task `here instead of topic partitions list as I did before - and this makes sense, because the public `ICommittable` interface returns general `Task` without typed value.

Reply:
I think that's fine